python-packages/fle_utils/django_utils/command.py

```diff
@@ -131,7 +131,15 @@ def call_command_async(cmd, *args, **kwargs):
     that stringify in a way that commands can parse
     (which will work for str, bool, int, etc).
     """
+    #  Workaround for #3704. It seems like running cron for downloading videos
+    # as a different thread on the same process in OS X results in a normal exit.
+    # That's it, the kalite executable just quits, no segfaults or anything.
+    # The workaround for now is to spawn subprocesses by default on OS X,
+    # and have everyone else spawn threads.
+    is_osx = sys.platform == 'darwin'
+    in_proc = kwargs.pop('in_proc', not is_osx)
     in_proc = kwargs.pop('in_proc', True)
```

Comment:
I commented this line-out and the other process didn't crash.  The `/progress/` API call is still working but with 0 bytes being downloaded, it's only a 1.3MB video.

Testing it further...